app/src/main/java/com/kickstarter/viewmodels/CommentsViewHolderViewModel.kt

```diff
@@ -171,15 +181,32 @@ interface CommentsViewHolderViewModel {
                 .subscribe(this.flagComment)
         }
 
-        private fun isActionGroupVisible(commentCardData: CommentCardData, user: User?) =
-            commentCardData.project?.let {
-                it.isBacking || ProjectUtils.userIsCreator(it, user)
```

Comment:
isCreator was affecting the flagging button that has been deleted, so `ProjectUtils.userIsCreator(it, user)` needed to be removed as well

Reply:
nevermind, `ProjectUtils.userIsCreator(it, user)` needs to be back as creators must see the reply button